images/metering-ansible-operator/roles/meteringconfig/tasks/configure_tls.yml

```diff
@@ -0,0 +1,60 @@
+---
+
+- name: Check if we need to create a Presto TLS secret using the user-populated fields
```

Comment:
The name here doesn't match the logic. It's just doing validation on the Presto Server TLS configuration.